docs/contributor/02-10-testing-strategy.md

```diff
@@ -0,0 +1,85 @@
+# Testing strategy of the Istio module
+
+## Background
+The Istio module requires a testing strategy to ensure that the module is always functioning
+as expected on all supported platforms.
+However, running tests on all supported platforms in all cases is both time-consuming and expensive.
+Therefore, we need to decide on a testing strategy that balances the need for comprehensive
+testing with the requirement for fast feedback and development.
+
+## Testing strategy outline
+
+The testing strategy for the Istio module will be implemented according to the following guidelines:
+1. Tests that depend on the Gardener platform will not run on Pull Requests (PRs).
```

Comment:
```suggestion
1. Tests that depend on the Gardener platform do not run on Pull Requests (PRs).
```
If I understand correctly, it's a testing strategy located in the contributor folder (a set of rules that you want to follow and update if needed). So, we can use a present tense